Library/Formula/saltstack.rb

```diff
@@ -28,9 +27,9 @@ class Saltstack < Formula
   # Homebrew's swig breaks M2Crypto due to upstream's undermaintained status.
   # https://github.com/swig/swig/issues/344
   # https://github.com/martinpaljak/M2Crypto/issues/60
-  resource "swig304" do
-    url "https://downloads.sourceforge.net/project/swig/swig/swig-3.0.4/swig-3.0.4.tar.gz"
-    sha256 "410ffa80ef5535244b500933d70c1b65206333b546ca5a6c89373afb65413795"
+  resource "swig307" do
```

Comment:
You'll need to revert this, I'm afraid. The original problem is still there because M2Crypto haven't committed any fix upstream:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/opt/saltstack/libexec/vendor/lib/python2.7/site-packages/M2Crypto/__init__.py", line 24, in <module>
    import ASN1
  File "/usr/local/opt/saltstack/libexec/vendor/lib/python2.7/site-packages/M2Crypto/ASN1.py", line 12, in <module>
    import BIO
  File "/usr/local/opt/saltstack/libexec/vendor/lib/python2.7/site-packages/M2Crypto/BIO.py", line 221, in <module>
    class CipherStream(BIO):
  File "/usr/local/opt/saltstack/libexec/vendor/lib/python2.7/site-packages/M2Crypto/BIO.py", line 227, in CipherStream
    SALT_LEN = m2.PKCS5_SALT_LEN
AttributeError: 'module' object has no attribute 'PKCS5_SALT_LEN'
```